src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlQueueClient.cs

```diff
@@ -277,5 +284,12 @@ private static void PopulateGetJobsCommand(SqlCommand cmd, byte queueType, long?
                 cmd.Parameters.AddWithValue("@ReturnDefinition", returnDefinition.Value);
             }
         }
+
+        private static void PopulateGetActiveJobsCommand(SqlCommand cmd, byte queueType)
```

Comment:
I am not sure why this ius a separate method. Looks like it is called from single place?